ci_tests/cfg-indent-check.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import os
+import re
+import sys
+
+
+def cfg_indention_check():
+    exit_code = 0
+    for file in sys.argv[1:]:
+        status_code = 0
+        cfg_path = os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                                os.path.pardir, file))
```

Comment:
A nitpick: according to the [PyDoc](https://docs.python.org/3.10/library/os.html?highlight=pardir#os.pardir) `os.pardir` is the public reference and should be used instead of `os.path.pardir`.